### ISSUE TITLE
Can't get GUI sometime

### ISSUE THREAD

**Issue by 0312birdzhang** [sailfish-on-davinci/issues#12]
Reboot to recovery and then reboot can fix this, odd.

**Comment by 0312birdzhang** [comment]
Reproduced, but don't know how to trigger it.

**Comment by 0312birdzhang** [comment]
[dmesg.log](https://github.com/sailfish-on-davinci/issues/files/4958261/d1.log)
[journalctl.log](https://github.com/sailfish-on-davinci/issues/files/4958264/j1.log)
[logcat.log](https://github.com/sailfish-on-davinci/issues/files/4958265/logcat.log)

**Comment by 0312birdzhang** [comment]
Reproduced agagin:
```
[root@RedmiK20 nemo]# journalctl |grep lipstick
Aug 05 13:43:32 RedmiK20 systemd[4026]: Starting The lipstick UI...
Aug 05 13:43:33 RedmiK20 lipstick[4634]: == hwcomposer module ==
Aug 05 13:43:33 RedmiK20 lipstick[4634]: * Address: 0xe0991004
Aug 05 13:43:33 RedmiK20 lipstick[4634]: * Module API Version: 3
Aug 05 13:43:33 RedmiK20 lipstick[4634]: * HAL API Version: 0
Aug 05 13:43:33 RedmiK20 lipstick[4634]: * Identifier: hwcomposer
Aug 05 13:43:33 RedmiK20 lipstick[4634]: * Name: QTI Hardware Composer Module
Aug 05 13:43:33 RedmiK20 lipstick[4634]: * Author: CodeAurora Forum
Aug 05 13:43:33 RedmiK20 lipstick[4634]: == hwcomposer module ==
Aug 05 13:43:34 RedmiK20 lipstick[4634]: == hwcomposer device ==
Aug 05 13:43:34 RedmiK20 lipstick[4634]: * Version: 2000001 (interpreted as 2000001)
Aug 05 13:43:34 RedmiK20 lipstick[4634]: * Module: 0xe0991004
Aug 05 13:43:34 RedmiK20 lipstick[4634]: == hwcomposer device ==
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [D] unknown:0 - onHotplugReceived(0, 0, connected, primary)
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [W] unknown:0 - EGLFS: Failed to open /dev/fb0
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [D] unknown:0 - EGLFS: Screen Info
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [D] unknown:0 -  - Physical size: QSizeF(68.0695, 147.119)
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [D] unknown:0 -  - Screen size: QSize(1080, 2340)
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [D] unknown:0 -  - Screen depth: 32
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [W] unknown:0 - QEglScreen 0x9eab80
Aug 05 13:43:34 RedmiK20 lipstick[4634]: [W] unknown:0 - Notifications restored: 0
Aug 05 13:43:36 RedmiK20 lipstick[4634]: [D] unknown:0 - Hardware Compositor support is disabled
Aug 05 13:43:36 RedmiK20 lipstick[4634]: [W] unknown:0 - QFileSystemWatcher::addPaths: list is empty
Aug 05 13:43:36 RedmiK20 lipstick[4634]: [W] unknown:51 - file:///usr/share/lipstick-jolla-home-qt5/layers/Layer.qml:51:43: Unable to assign [undefined] to bool
Aug 05 13:43:36 RedmiK20 lipstick[4634]: [W] unknown:98 - file:///usr/share/lipstick-jolla-home-qt5/layers/Layer.qml:98:25: Unable to assign [undefined] to float
Aug 05 13:43:36 RedmiK20 lipstick[4634]: [D] unknown:0 - Registered VoiceCallAgent /com/jolla/Voicecall
Aug 05 13:43:36 RedmiK20 dbus-daemon[4338]: dbus-daemon[4338]: [session uid=100000 pid=4338] Activating via systemd: service name='org.bluez.obex' unit='dbus-org.bluez.obex.service' requested by ':1.21' (uid=100000 pid=4634 comm="/usr/bin/lipstick -plugin evdevtouch -plugin evdev")
Aug 05 13:43:36 RedmiK20 lipstick[4634]: QInotifyFileSystemWatcherEngine::addPaths: inotify_add_watch failed: No such file or directory
Aug 05 13:43:37 RedmiK20 mce[2239]: tklock.c: tklock_dbus_notification_end_cb(): notification end from name=:1.44 owner=:1.44 pid=4634 uid=100000 gid=996 priv=1 cmd=/usr/bin/lipstick -plugin evdevtouch -plugin evdevkeyboard:keym
Aug 05 13:43:37 RedmiK20 dbus-daemon[4338]: dbus-daemon[4338]: [session uid=100000 pid=4338] Activating via systemd: service name='org.maliit.server' unit='maliit-server.service' requested by ':1.21' (uid=100000 pid=4634 comm="/usr/bin/lipstick -plugin evdevtouch -plugin evdev")
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:18 - file:///usr/share/lipstick-jolla-home-qt5/notifications/NotificationPreview.qml:18:1: QML NotificationPreview: Binding loop detected for property "notification"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:18 - file:///usr/share/lipstick-jolla-home-qt5/notifications/NotificationPreview.qml:18:1: QML NotificationPreview: Binding loop detected for property "notification"
Aug 05 13:43:37 RedmiK20 mce[2239]: tklock.c: tklock_dbus_notification_beg_cb(): notification begin from name=:1.44 owner=:1.44 pid=4634 uid=100000 gid=996 priv=1 cmd=/usr/bin/lipstick -plugin evdevtouch -plugin evdevkeyboard:keym
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:74 - file:///usr/lib/qt5/qml/Sailfish/Silica/BackgroundItem.qml:74:5: QML : Binding loop detected for property "width"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:74 - file:///usr/lib/qt5/qml/Sailfish/Silica/BackgroundItem.qml:74:5: QML : Binding loop detected for property "width"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:74 - file:///usr/lib/qt5/qml/Sailfish/Silica/BackgroundItem.qml:74:5: QML : Binding loop detected for property "width"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:74 - file:///usr/lib/qt5/qml/Sailfish/Silica/BackgroundItem.qml:74:5: QML : Binding loop detected for property "width"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:18 - file:///usr/share/lipstick-jolla-home-qt5/notifications/NotificationPreview.qml:18:1: QML NotificationPreview: Binding loop detected for property "notification"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:18 - file:///usr/share/lipstick-jolla-home-qt5/notifications/NotificationPreview.qml:18:1: QML NotificationPreview: Binding loop detected for property "notification"
Aug 05 13:43:37 RedmiK20 mce[2239]: tklock.c: tklock_dbus_notification_beg_cb(): notification begin from name=:1.44 owner=:1.44 pid=4634 uid=100000 gid=996 priv=1 cmd=/usr/bin/lipstick -plugin evdevtouch -plugin evdevkeyboard:keym
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - "Method \"GetProperties\" with signature \"\" on interface \"net.connman.Technology\" doesn't exist\n"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - QEglWindow 0x1232628: 0xb84c80 0x0
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [W] unknown:0 - libpng warning: iCCP: known incorrect sRGB profile
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 systemd[4026]: Started The lipstick UI.
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - Registered Bluetooth OBEX agent: "/com/jolla/obexservice/agent"
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
Aug 05 13:43:37 RedmiK20 lipstick[4634]: [D] unknown:0 - prepare: validate failed for display 0: 2
```

**Comment by 0312birdzhang** [comment]
> 
> 
> Reboot to recovery and then reboot can fix this, odd.

It's not working now :(

**Comment by 0312birdzhang** [comment]
Seems fix with 3.4.0, not sure this helpful or not https://github.com/sailfish-on-davinci/droid-config-davinci/commit/220dbd32fc0a353c9a45fc8415db1c3c561b5db1
